.github/CODEOWNERS

```diff
@@ -2,6 +2,7 @@
 
 # Core
 common/djangoapps/student/
+common/djangoapps/student/models.py               @openedx/teaching-and-learning
```

Comment:
The syntax for this change looks correct, but I do not believe it will have the effects you intend it to. On GitHub, there will be certain "owner icons" that will appear, and hovering over those will indicate that this file is owned by TNL. Beyond this, when individuals create PRs on this file, TNL will be suggested as reviewers. Finally, if the repo's approval rules are modified to require approval by a code owner, then merging would be blocked without approval by one of us. This was the feature underpinning our former proposed "band aid" solution. We are no longer proposing to do this. In short, I do not believe that this modification to the CODEOWNERS file will generate the desired notification.

Reply:
Something along the lines of [this article](https://stackoverflow.com/questions/19169787/github-file-change-notification) may be more fruitful